iree/compiler/Dialect/Shape/Plugins/XLA/XlaHloShapeBuilder.cpp

```diff
@@ -283,6 +283,38 @@ Value rewriteDynamicReshape(RankedShapeType resultShape, DynamicReshapeOp op,
                                             op.output_shape());
 }
 
+Value rewriteSelectOp(RankedShapeType resultShape, SelectOp selectOp,
```

Comment:
you may add a ut for select and compare op  with dynamic shape tensor as input ,like the one we have in iree/test/e2e/regression/dynamic_add.mlir

Reply:
done

Reply:
> you may add a ut for select and compare op with dynamic shape tensor as input ,like the one we have in iree/test/e2e/regression/dynamic_add.mlir

done